instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './SearchBar.css';
+
```

Comment:
Once you finish Day 4, try to work on adding functionality to the search bar and the likes